code/src/index.css

```diff
@@ -11,3 +11,165 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+header {
+  height: 100px;
+  text-align: center;
+  color: rgb(244, 111, 133);
+  font-family: "lobster", sans-serif;
+}
+
+p {
+  font-family: "Trispace", sans-serif;
+  word-wrap: break-word;
+}
+
+.letter-counter {
+  font-size: 12px;
+}
+
+.date {
+  color: rgb(165, 165, 165);
+  float: right;
+  margin: 20px 0px;
+  font-size: 12px;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 40px;
+}
+
+.thoughts-wrapper,
+.submit-wrapper {
+  border: 1px solid black;
+  width: 320px;
+  padding: 5px 10px;
+  box-shadow: 6px 6px #000;
+  background: white;
+}
+
+.submit-wrapper {
+  display: flex;
+  background: rgb(228, 228, 228);
+}
+
+textarea {
+  width: 310px;
+  margin-bottom: 10px;
+}
+
+.like-btn {
+  border: none;
+  background: pink;
+  border-radius: 50%;
+  padding: 2%;
+  width: 40px;
+  margin: 10px 6px 0px 0px;
+}
+
+.like-heart {
```

Comment:
This was my main issue, to get the hearts in the center.
They are still a bit to the right when I look in the browser (chrome).
I ended up adding a break-space `&nbsp`(not perfect still)